public/app/plugins/panel/live/module.tsx

```diff
@@ -1,14 +1,37 @@
 import { PanelPlugin } from '@grafana/data';
 import { LiveChannelEditor } from './LiveChannelEditor';
 import { LivePanel } from './LivePanel';
-import { LivePanelOptions } from './types';
+import { LivePanelOptions, MessageDisplayMode } from './types';
 
 export const plugin = new PanelPlugin<LivePanelOptions>(LivePanel).setPanelOptions(builder => {
   builder.addCustomEditor({
+    category: ['Channel'],
     id: 'channel',
     path: 'channel',
     name: 'Channel',
     editor: LiveChannelEditor,
     defaultValue: {},
   });
+
+  builder
+    .addRadio({
+      path: 'message',
+      name: 'Show Message',
+      description: 'Display the last message recieved on this channel',
+      settings: {
+        options: [
+          { value: MessageDisplayMode.Raw, label: 'Raw Text' },
+          { value: MessageDisplayMode.JSON, label: 'JSON' },
+          { value: MessageDisplayMode.Auto, label: 'Auto' },
+          { value: MessageDisplayMode.None, label: 'None' },
+        ],
+      },
+      defaultValue: MessageDisplayMode.JSON,
+    })
+    .addBooleanSwitch({
+      path: 'publish',
+      name: 'Show Publish',
+      description: 'Display the a form to publish values',
```

Comment:
Not sure which is meant here - `Display the form to publish values` / `Display a form to publish values`?

Reply:
dooh -- either, but not both!